src/api/core.js

```diff
@@ -137,8 +137,8 @@ export const parseEndpoints = data => {
 				const { group, description } = guessEndpointDocumentation( method, data.namespace, pathLabel );
 
 				const endpoint = {
-					path_format: pathFormat || '/',
-					path_labeled: pathLabel || '/',
+					pathFormat: pathFormat || '/',
+					pathLabeled: pathLabel || '/',
```

Comment:
I think, you'll need to change this on the parsing of endpoints for .com API as well (these names have been written like that at first to match the discovery of .com endpoints)